examples/singleton/README.md

```diff
@@ -0,0 +1,23 @@
+# Auth filter example
+
+You can run the example like so:
+
+```
+npm run asbuild
+podman run -ti --rm -p 8080:8080 -p 8001:8001 --entrypoint=envoy -v $PWD/envoycfg.yaml:$PWD/envoycfg.yaml:ro -v $PWD/build:$PWD/build:ro -w $PWD docker.io/envoyproxy/envoy-dev:f6679d51cc7b2f0b5e05c883a035ad87d011f454 -c $PWD/envoycfg.yaml
```

Comment:
Do we need this particular sha of `envoy-dev`? Will this work with `v1.17.0`? Would be nice to specify a version for context if anyone's looking at this in the future after a few more Envoy releases.

Reply:
it won't unfortunately; my patch was merged after this release